tests/pds/restore_test.go

```diff
@@ -103,3 +105,195 @@ var _ = Describe("{PerformRestoreToSameCluster}", func() {
+					log.FailOnError(err, "Failed while performing adhoc backup.")
+				})
+				stepLog = "Perform multiple restores for the backup jobs in parallel."
+				Step(stepLog, func() {
+					log.InfoD(stepLog)
+					numberOfIterations := 3
+					var wg sync.WaitGroup
+					for i := 0; i < numberOfIterations; i++ {
+						log.Infof("Triggering restore operation")
+						wg.Add(1)
+						go func() {
+							defer wg.Done()
+							defer GinkgoRecover()
+							ctx := pdslib.GetAndExpectStringEnvVar("PDS_RESTORE_TARGET_CLUSTER")
+							restoreTarget := tc.NewTargetCluster(ctx)
+							restoreClient := restoreBkp.RestoreClient{
+								TenantId:             tenantID,
+								ProjectId:            projectID,
+								Components:           components,
+								Deployment:           deployment,
+								RestoreTargetCluster: restoreTarget,
+							}
+							backupJobs, err := restoreClient.Components.BackupJob.ListBackupJobsBelongToDeployment(projectID, deployment.GetId())
+							log.FailOnError(err, "Error while fetching the backup jobs for the deployment: %v", deployment.GetClusterResourceName())
+							for _, backupJob := range backupJobs {
+								log.Infof("[Restoring] Details Backup job name- %v, Id- %v", backupJob.GetName(), backupJob.GetId())
+								restoredModel, err := restoreClient.TriggerAndValidateRestore(backupJob.GetId(), params.InfraToTest.Namespace, dsEntity, false, true)
+								log.FailOnError(err, "Failed during restore.")
+								restoredDeployment, err = restoreClient.Components.DataServiceDeployment.GetDeployment(restoredModel.GetDeploymentId())
+								log.FailOnError(err, fmt.Sprintf("Failed while fetching the restore data service instance: %v", restoredModel.GetClusterResourceName()))
+								deploymentsToBeCleaned = append(deploymentsToBeCleaned, restoredDeployment)
+								log.InfoD("Restored as Deployment: %v successfully!", restoredModel.GetClusterResourceName())
+							}
+						}()
+					}
+					wg.Wait()
+					log.Infof("Restoring multiple backup job succeeded.")
+				})
+
+				Step("Delete Deployments", func() {
+					CleanupDeployments(deploymentsToBeCleaned)
+				})
+			}
+		})
+	})
+	JustAfterEach(func() {
+		defer EndTorpedoTest()
+		err := bkpClient.AWSStorageClient.DeleteBucket()
+		log.FailOnError(err, "Failed while deleting the bucket")
+	})
+})
+
+var _ = Describe("{PerformSimultaneousRestoresDifferentDataService}", func() {
+	bkpTargetName = bkpTargetName + pdsbkp.RandString(8)
+	JustBeforeEach(func() {
+		StartTorpedoTest("PerformSimultaneousBackupRestoreForMultipleDeployments", "Perform multiple backup and restore simultaneously for different deployments.", pdsLabels, 0)
+		bkpClient, err = pdsbkp.InitializePdsBackup()
+		log.FailOnError(err, "Failed to initialize backup for pds.")
+		bkpTarget, err = bkpClient.CreateAwsS3BackupCredsAndTarget(tenantID, fmt.Sprintf("%v-aws", bkpTargetName), deploymentTargetID)
```

Comment:
same as above, we need to clean up backup targets

Reply:
Automation is ready, blocked due to following bug: https://portworx.atlassian.net/browse/DS-5981